content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/index.mdx

```diff
@@ -32,6 +32,13 @@ In order to successfully complete this guide, you need:
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'planetscale']}>
+
+- [Node.js](https://nodejs.org/en/) installed on your machine
+- a [PlanetScale](https://www.planetscale.com/) database server running
```

Comment:
I would suggest emphasizing that the `main` branch should not be promoted to production.

Reply:
Thanks! To check I understand: are you thinking of something on the lines of 'This tutorial will assume that you can push to the `main` branch of your database. Do not do this if your `main` branch has been promoted to production.'?

Reply:
Exactly! By default, the `main` branch isn't a production branch when your first create the database.

Reply:
Added this in.